CMakeLists.txt

```diff
@@ -520,6 +520,12 @@ if (ISLE_BUILD_APP)
     target_compile_definitions(isle PRIVATE "ISLE_EMSCRIPTEN_HOST=\"${ISLE_EMSCRIPTEN_HOST}\"")
     set_property(TARGET isle PROPERTY SUFFIX ".html")
   endif()
+  if(NINTENDO_3DS)
+    target_sources(isle PRIVATE
+      ISLE/3ds/apthooks.cpp
+      ISLE/3ds/config.cpp
+    )
+  endif()
 endif()
```

Comment:
`install(TARGETS isle)` will probably install a elf file instead of a common `.3ds` file,
so you might want to conditionally disable these lines, and install the `.3ds` file instead.